services/ui-src/src/containers/HomeState.js

```diff
@@ -1,29 +1,53 @@
 import React from "react";
 import { Accordion, Grid, Link } from "@trussworks/react-uswds";
+import { connect } from "react-redux";
 
-const HomeState = () => {
-  // TODO: Pull state from redux
-  const state = "AL";
+const HomeState = ({ userData }) => {
+  // Pull state from redux
+  const state = userData.abbrev;
```

Comment:
${state} is not being populated in the link when you click on the Quarter you want to view. Could you follow up to see why? In LocalLogin.js I changed the info for Alice Cooper to be role: "state" and states: ["AL"] only so it should have picked up Alabama and populated that when moving from this page to the Quarter page.